src/Kernel/IQSharpEngine.cs

```diff
@@ -75,9 +79,71 @@ IMagicSymbolResolver magicSymbolResolver
                 Process.GetCurrentProcess().Id
             );
 
+
             eventService?.TriggerServiceInitialized<IExecutionEngine>(this);
         }
 
+        /// <summary>
+        ///     Registers an event handler that searches newly loaded packages
+        ///     for extensions to this engine (in particular, for result encoders).
+        /// </summary>
+        private void RegisterPackageLoadedEvent(IServiceProvider services, ILogger logger, IReferences references)
+        {
+            var knownAssemblies = references
+                .Assemblies
+                .Select(asm => asm.Assembly.GetName())
+                .ToImmutableHashSet()
+                // Except assemblies known at compile-time as well.
+                .Add(typeof(StateVectorToHtmlResultEncoder).Assembly.GetName());
```

Comment:
When #177 is merged, the AzureClient assembly will also have some display encoders in it. Just FYI to whichever one of us merges second.

Reply:
Good to know, thanks! Given what you said about conda breaking until #177, it probably makes sense for that one to go first and for me to catch up?

Reply:
Probably yes, that makes sense. Hoping to be able to merge that first thing tomorrow morning.